<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Newplan</class>
 <widget class="QMainWindow" name="Newplan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>720</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>701</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lb_newplan">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>child_newplan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_nr">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>等线 Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>- 计划内容</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_sj">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>等线 Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>- 计划起止时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDateTimeEdit" name="dte1">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>等线 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>等线 Light</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>至</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dte2">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>等线 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbtn_yes">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
